docs/content/integrations/duckdb/using-duckdb-with-dagster.mdx

```diff
@@ -30,45 +43,196 @@ To complete this tutorial, you'll need:
 
 ---
 
-## Step 1: Configure the DuckDB I/O manager
+## Part 1: Using the DuckDB resource
+
+---
+
+### Step 1: Configure the DuckDB resource
```

Comment:
```suggestion
## Part 1: Using the DuckDB resource

### Step 1: Configure the DuckDB resource
```

Sorry if one of my suggestions did this - rules are only needed above H2 headings.